sdk/nodejs/runtime/dagger.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "typescript-sdk",
+  "sdk": "go",
+  "root": "..",
+  "sdkRuntime": "vito/dagger-sdk-go:f239c6f6c3ceaa1f760b589fac43b1d0704e45d3@sha256:915f41b414b12a878c60d96869f8ff8155e6ce122b1849303b995b65a343591f"
```

Comment:
This is outdated, no longer used.
```suggestion
```

Reply:
It's still in the configuration though

Reply:
How so? If you dagger mod init it shouldn't be there so where did it come from? Did you have it already created from an earlier attempt?

Reply:
That's right, strange, fixed in [a1c61d9](https://github.com/dagger/dagger/pull/6220/commits/a1c61d968be22f8dee5d74b590d8938a6d0c9fb6)